v2/car.go

```diff
@@ -0,0 +1,70 @@
+package car
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+const HeaderBytesLen uint64 = 32
+
+var (
+	// The fixed prefix of a CAR v2, signalling the version number to previois versoions for graceful fail over.
+	PrefixBytes = []byte{
```

Comment:
probably best to not export these vars

Reply:
this becomes the magic bytes for filetype detection, so there's a plausible reason another package might want to reference them.